doc-examples/example-groovy/src/main/groovy/example/ProfileService.groovy

```diff
@@ -45,7 +43,7 @@ class ProfileService {
             } else {
                 return Optional.empty()
             }
-        } catch (ObjectStorageException | IOException e) {
+        } catch (IOException e) {
```

Comment:
we shouldn't encourage readers to squash exceptions like this. either re-throw, or throw a new exception with the IOException as the cause, or even log the error. we're trying to make developers more productive, not less